playground/components/header.tsx

```diff
@@ -62,90 +72,146 @@ export const Header: Component<{
+            </button>
+
+            <button
+              type="button"
+              onClick={() => exportToCsb(props.tabs)}
+              class="text-black md:text-white flex flex-row space-x-2 items-center md:px-3 px-2 py-2 focus:outline-none focus:ring-1 rounded text-white opacity-80 hover:opacity-100"
+              title="Export to CodeSandbox"
+            >
+              <svg
+                class="fill-current h-6 px-[1.63px] py-0"
+                preserveAspectRatio="xMidYMid"
+                viewBox="0 0 256 296"
+              >
+                <path d="M115 261V154l-91-52v61l42 24v46l49 28zm24 1l51-29v-47l42-25v-60l-93 54v107zm81-181l-49-28-43 24-43-24-49 28 92 53 92-53zM0 222V74L128 0l128 74v148l-128 74L0 222z" />
+              </svg>
+              <span class=" text-xs md:sr-only">Export to CodeSandbox</span>
+            </button>
+
+            <button
+              type="button"
+              onClick={shareLink}
+              class="text-black md:text-white flex flex-row space-x-2 items-center md:px-3 px-2 py-2 focus:outline-none focus:ring-1 rounded"
+              classList={{
+                'opacity-80 hover:opacity-100': !copy(),
+                'text-green-100': copy() && !showMenu(),
+              }}
+              title="Share with a minified link"
+            >
+              <Icon class="h-6" path={copy() ? link : share} />
+              <span class="text-xs md:sr-only">{copy() ? 'Copied to clipboard' : 'Share'}</span>
+            </button>
+          </div>
+        </div>
         <button
           type="button"
-          onClick={props.toggleDark}
-          class="px-3 py-2 focus:outline-none focus:ring-1 rounded text-white opacity-80 hover:opacity-100"
-          title="Toggle dark mode"
+          id="menu-btn"
+          class="z-40 px-3 py-2 focus:outline-none focus:ring-1 rounded text-white opacity-80 hover:opacity-100 visible relative md:hidden m-0 mr-[10px]"
+          classList={{ 'border-white border': showMenu() }}
+          title="Mobile Menu Button"
+          onClick={() => setShowMenu(!showMenu())}
         >
-          <span class="sr-only">{props.dark ? 'Light' : 'Dark'} mode</span>
-
-          <Show
-            when={props.dark}
-            fallback={
-              <svg class="h-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-                <path
-                  stroke-linecap="round"
-                  stroke-linejoin="round"
-                  stroke-width="2"
-                  d="M20.354 15.354A9 9 0 018.646 3.646 9.003 9.003 0 0012 21a9.003 9.003 0 008.354-5.646z"
-                />
-              </svg>
-            }
-          >
-            <svg class="h-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+          {showMenu() ? (
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              class="h-6 w-6"
+              fill="none"
+              viewBox="0 0 24 24"
+              stroke="currentColor"
+            >
```

Comment:
We can replace the whole SVG by the icon from `@amoutonbrady/solid-heroicons`:
```suggestion
            <Icon path={xCircle} class="h-6 w-6" />
```

Reply:
Done